### PR TITLE
code refactored to fetch two cities and compare them

### PR DESCRIPTION
**Pull request by mhidalgo83**
Fixes #  :computer:

Description of changes in this pull request:
- refactored SearchBar component, bringing business logic to parent component
- SearchPanel component includes sliders and extra search bar
- SearchPanel pushes to Comparison component with data from backend

Testing Status: Not tested

Reviewed By: 

@Lambda-School-Labs/citrics-fe-a

### CODE REVIEW

File: src/components/common/SearchBar/Component.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { useHistory, useLocation } from 'react-router-dom';
 import '../../../styles/style.less';
 
 function SearchBar(props) {
+  console.log(props.newProps);
```
Comment (kc0buk) [inline src/components/common/SearchBar/Component.js]: Should we remove console logs before pushing to main @mhidalgo83 ?

File: src/components/common/SearchPanel/Component.js [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
 
 export default ({ display }) => {
+  const { fetchSpecificCityData } = cardContainerActs;
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const currentLocation = useLocation().pathname;
+  const [cities, setCities] = useState([]);
+
+  useEffect(() => {
+    console.log(cities);
```
Comment (kc0buk) [inline src/components/common/SearchPanel/Component.js]: Remove console log here as well?

Comment (mhidalgo83) [reply src/components/common/SearchPanel/Component.js]: Yeah, I can get rid of those.

File: src/components/common/SearchBar/Component.js (hunk 1 above)
Comment (mhidalgo83) [reply src/components/common/SearchBar/Component.js]: Yeah.

**Review by adonispuente — APPROVED** [review]
It was more than one file but the changes in most were small (moving props around). Good PR overall.

**Review by kc0buk — APPROVED** [review]
Nice job figuring out how to get props passed between components!

**Pull request merged**